be/src/exprs/bloom_filter_func.h

```diff
@@ -244,6 +244,7 @@ class BloomFilterFuncBase : public RuntimeFilterFuncBase {
         _bloom_filter = other_func->_bloom_filter;
         _inited = other_func->_inited;
         _enable_fixed_len_to_uint32_v2 |= other_func->_enable_fixed_len_to_uint32_v2;
+        set_filter_id(bloomfilter_func->get_filter_id());
```

Comment:
Why reset filter id here? Should we set it during initialization?

Reply:
![QQ_1736908064276](https://github.com/user-attachments/assets/4f127c0b-e0e9-4f70-94d5-14e469e846c3)
We don't have rf desc here so copy filter id is needed